### DISCUSSION TITLE
loan request shopping cart

### DISCUSSION THREAD

**Discussion by dustymc** [ArctosDB/arctos#4917] [General]
ref: https://github.com/ArctosDB/arctos/issues/1875

Should we let users request specific parts for loan?

**Comment by campmlc** [comment]
Yes. And we should consider adding the same information we provide to GGBN
regarding permits for genomic materials so that Arctos can be a searchable
alternative to the GGBN interface.

**Comment by KyndallH** [comment]
Yes! Not just for tissues but skeletal too. How would that work when we enter the skeleton and all they need is the right femur? 

I like having all the permits tied in too. 

Would this just be selecting the part they need or can we incorporate the whole loan request process? Could it be tailored for the collection? Could collections that haven't warmed to the idea, opt out?

**Comment by dustymc** [comment]
>permits

I think that's deserving of a separate Issue - not much technical problem, but exposing permit data (or a lack thereof) is likely a sensitive issue that should be clearly presented to the community without any other clutter.

>tissues but skeletal too

You will never see me trying to make a preemptive distinction between tissues and "vouchers!"

>How would that work when we enter the skeleton and all they need is the right femur?

You tell me, but I suspect this would still involve a fair bit of interaction - eg, they'd enter a comment somewhere, you'd probably want to contact them personally before hucking a bunch of stuff in a box, etc.

>incorporate the whole loan request process?

I think that would be very useful. And this would likely spawn cross-collection requests, so I think it would be based on a project to tie them all together.

>tailored for the collection?

Probably in a limited capacity - I don't think we have the resources to maintain code for every collection, but the documents involved and such would definitely be linked to specific collections.

>opt out

Yea that's probably necessary - no reason to have people fill in forms if the collections aren't willing to accept them.

**Comment by campmlc** [comment]
Regarding permits, we have already presented this to the community in
relation to tissues because of GGBN, and this permit information is going
to be online hopefully very soon. No reason not to include what we already
make available. But perhaps an opt-in button to show permit availability at
the discretion of the collection, so that other types of collections can
choose to add this funcitonality if they want?

**Comment by dustymc** [comment]
>GGBN

That's a very different system from which one can ask very different questions of very different data; I don't think it directly applies to "Arctos Proper."

**Comment by campmlc** [comment]
Because of the UN Convention on Biological Diversity Nagoya Protocol, any
tissues collected after 2012 are required to show permits. That means
Arctos collections are affected, whether or not we use GGBN. We have to be
able to show potential loan recipients that our tissues have permits. I
think it is important that we can do this in Arctos, so I don't have to
tell researchers from the EU to go to the GGBN portal instead.

**Comment by KyndallH** [comment]
The US currently is not part of the Nagoya Protocol, right? So it is only if we deal with one of the 100 something countries that are in it. And how much of the permit do they need? I'm wondering if we should do a refresher for everyone on it because I figure one day, the US will join. Nice to be ahead of the curve when they do join. Beyond the Nagoya Protocol, we should have everything associated with permits and be able to track it.

**Comment by dustymc** [comment]
No US isn't in Nagoya.

If you can find tissues with permits, then some bored intern can find things that should but do not have permits. (Eg all federal lands have required collecting permits since 1906.) I'm happy to open that door, but from the middle of some unrelated Issue that the affected people aren't likely to see.

**Comment by campmlc** [comment]
The EU and Canada are signatories, as are most/many of the developing
countries we have collections from. So if a Canadian or EU researcher wants
a loan of Ecuadorian tissues from MSB, they are required to verify whether
the tissues were collected in according with Nagoya, e.g. with access and
benefits sharing agreements with the source country. So even if the US is
not a signatory, since we house tissues from countries that are, and since
we loan to countries that are, we are affected by it and need to document
that the tissues were collected in accordance with the protocol.

**Comment by dustymc** [comment]
Tabling for apparent lack of interest.

**Comment by campmlc** [comment]
Not lack of interest, just got buried. This would actually be hugely helpful.

**Comment by Jegelewicz** [comment]
Discussed in Object Tracking planning today - any chance we can create a mock-up? How different does this need to be from the internal loan form?

**Comment by campmlc** [comment]
It would need to allow agent data from new agents that don't exist yet in
Arctos, e.g. provide all info to create the agent if the loan is approved.
It would need to provide all the info needed to create a project in Arctos.
It would need to allow specific fields that could be selected by the
collection - for example, a genomics loan request form could be customized
to request tissue type, quantity, and preservation.
Ideally allow for requesting specific catalog records, with alternates
designated.
It would need to allow billing info for the shipment - e.g. a Fedex account
- in a format that would not be saved in Arctos? Is that even possible? I
currently don't fill out all the shipment info in Arctos because I don't
wan't Fedex account numbers saved there.
Notification contacts - who does the loan request go to?

**Comment by ccicero** [comment]
Sorry, I missed this thread. Personally I'm not in favor of this. There is often a lot of back and forth with requestors and I'm not convinced that this will simplify things. In fact, I expect it may make it more complicated. I also think there's something to be said for actually writing to the curators/collection managers in charge of collections with specifics of what they're requesting and why ("old school approach"), and I worry that some will just use this to choose whatever without really thinking about what they need for a project (e.g., 'give me everything of ....'). We had this discussion many years ago (pre-GitHub) when Link was head of the 'Steering Committee' and the general consensus at the time was NOT to do a 'shopping cart approach to loans. This needs a lot of discussion before any decisions are made. Has this ever come up at an issues meeting? Maybe we need a meeting focused just on loans and these inter-related issues?

**Comment by campmlc** [comment]
Yes, I think that makes sense, but we should probably have a discussion
especially as to how we are going to handle data exposed and requested via
third parties such as GGBN, and the permit exposure issue.
A related issue is a customizable Arctos form that collections can make
available for loan requests. We have strong need for this.

**Comment by dustymc** [comment]
>This needs a lot of discussion

Agree, that's why I transferred it here.

>issues meeting

Usage stats overwhelmingly suggest collections could learn from each other and I've been advocating some sort of 'how to loan' get-together forever, but I also think the issues meetings should be reserved for actionable (or nearly so) Issues. We're not making much headway now, adding another dimension isn't going to help. Maybe we need a new 'strategic planning' (or ??) committee?

>form that collections can make
available 

"Fill in the blanks and ...." is a workable loan policy - goferit, you don't need to wait on everyone else (which from here doesn't look imminent)

**Comment by ewommack** [comment]
> Maybe we need a new 'strategic planning' (or ??) committee?

For something as vital as loans are to collections, and the power behind Arctos with how we can handle loans, I wonder if we shouldn't try and schedule a symposium or something. Call it something where we say "hey we need a big discussion on this - it may change how we think about this vital thing - lets have a couple presentations on the methods we currently use and then talk about it"

**Comment by ccicero** [comment]
I like Beth's idea. 

Just for the record, I recently created a [form](https://docs.google.com/forms/d/e/1FAIpQLScXxZdbJINa71XXVRG4TTxw6zE06tgQ2vzPEPHhVLRD-3hWdA/viewform) for requesting loans as well as in-house visits (especially for ones where I create a loan), where people can upload their requests to a folder - cover letter, project description/justification, list of specimens, permits. Michelle set up a shared Google Drive folder for MVZ specifically, but maybe we can integrate that with Arctos? The list makes it easy to create a loan bulkloader file - although they usually don't include the part so that makes it complicated if there are multiple parts of the same type - e.g., multiple tissue samples. Perhaps some hybrid system where they can choose their parts which could then be used to bulkload the loan items, but they still have to go through this more formal request process?

I get notifications when a new request is submitted. My motivation was to avoid the abyss of email as much as possible, where requests tend to get lost and overlooked, as well as to minimize the back-and-forth or the time spent searching for a website to get a phone number. It's not perfect, but working much better than how I was dealing with requests before. It only took me 30 years to do this!

You have to go through the form to see everything, but it includes basic info including PI name/address/contact info, requestor - e.g., student - name/address/contact, funding info, project title and abstract, specifics of what they're requesting, whether it's a loan or in-house visit, shipment method if tissues (frozen, 95% ethanol, etc.), and other things that are needed but often omitted from requests. 

As new issues come up, it's easy for me to tweak the form.

**Reply by ewommack** [reply]
If we go down the symposium route, you could give a presentation showing your form system Carla. I've seen forms for other museums as well. @Jesyka15 I think you have one for Humboldt right?

**Comment by ccicero** [comment]
Sure. Happy to do that.

**Comment by dustymc** [comment]
>couple presentations on the method

I partially think that should be driven by citation stats, but I also suspect that everybody has SOMETHING that everyone else should be doing, and definitely would not want to use any sort of statistics to exclude anyone. 

I'm looking at Usage Statistics but there are a few views in reports.

<img width="798" alt="Screen Shot 2022-08-09 at 12 24 51 PM" src="https://user-images.githubusercontent.com/5720791/183744375-7308a7dd-d3b9-4209-b06b-6ebc86304761.png">

UAM Ento and Mamm loan a ridiculous amount of stuff, and most of it seems to get cited. 

MVZ:Egg has impressive ratios (87% loaned, 98% of that cited) for a smaller collection.


@ccicero how's that relate to your [loan policy](https://mvzhandbook.berkeley.edu/curatorial/loans-access/specimen-loans) (which get linked from guid-page headers) - I assume you manually send the form when it starts looking like they know what they're asking for??

**Comment by campmlc** [comment]
Reviving this. High priority for our institution to have public be able to request loan items by selectively adding items in search results to a saved search. This would mean adding to the ability to iteratively add items to an existing search from separate search results, so some sort of holding form or template would be necessary.
